packages/mongodb-chatbot-ui/src/InputBarTrigger.tsx

```diff
@@ -42,12 +46,15 @@ const styles = {
 };
 
 export type InputBarTriggerProps = DarkModeProps & {
+  bottomContent?: React.ReactNode;
+  fatalErrorMessage?: string;
+  placeholder?: string;
   suggestedPrompts?: string[];
 };
 
 export function InputBarTrigger(props: InputBarTriggerProps) {
   const { darkMode } = useDarkMode(props.darkMode);
-  const { suggestedPrompts = SUGGESTED_PROMPTS } = props;
+  const { suggestedPrompts = [], bottomContent, fatalErrorMessage = MongoDbChatbotFatalErrorMessage } = props;
```

Comment:
```suggestion
  const { suggestedPrompts = [], bottomContent, fatalErrorMessage = defaultChatbotFatalErrorMessage } = props;
```

per above suggestion